src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/JsonFlattener.cs

```diff
@@ -54,20 +56,22 @@ private void VisitJsonElement(JsonElement element)
                 case JsonValueKind.True:
                 case JsonValueKind.False:
                 case JsonValueKind.Null:
-                    _data.Add(new KeyValuePair<string, string>(_currentPath, element.ToString()));
+                    _data.Add(new KeyValuePair<string, string?>(_currentPath!, element.ToString()));
```

Comment:
`_currentPath` can be `null` here if we never execute `EnterContext` or `ExitContext`